scene/gui/graph_edit.cpp

```diff
@@ -2211,8 +2209,16 @@ void GraphEdit::remove_valid_left_disconnect_type(int p_type) {
 	valid_left_disconnect_types.erase(p_type);
 }
 
+void GraphEdit::set_connections(const TypedArray<Dictionary> &p_connections) {
+	clear_connections();
+
+	for (const Dictionary d : p_connections) {
```

Comment:
```suggestion
	for (const Dictionary &d : p_connections) {
```
Though it might not work with TypedArray (in which case leave it as is).

Reply:
It doesn't as far as I recall implementing it, binding to a cast breaks

Reply:
Although MSVC does not complain, other compilers do:
```
scene/gui/graph_edit.cpp:2262:25: error: loop variable 'd' of type 'const Dictionary &' binds to a temporary constructed from type 'const Variant &' [-Werror,-Wrange-loop-construct]
        for (const Dictionary &d : p_connections) {
                               ^
scene/gui/graph_edit.cpp:2262:7: note: use non-reference type 'Dictionary' to make construction explicit or type 'const Variant &' to prevent copying
        for (const Dictionary &d : p_connections) {
             ^~~~~~~~~~~~~~~~~~~~~
1 error generated.
```